canonical-web-and-design/ubuntu.com
# rename ecom analytics js file to please ublock

## Done

Rename the ecom analytics js file, because TIL uBlock will prevent loading of files that contain the word "analytics" in the filename.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- If you don't have it already, install the uBlock Origin extension on your browser ([Chrome](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwj397GwruvsAhXUa8AKHT6nBvUQFjAAegQIAxAC&url=https%3A%2F%2Fchrome.google.com%2Fwebstore%2Fdetail%2Fublock-origin%2Fcjpalhdlnbpafiamejdnhcphjbkeiagm%3Fhl%3Den&usg=AOvVaw05q6TgiJmPXMcHwEBAjzoF), [Firefox](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwj397GwruvsAhXUa8AKHT6nBvUQFjABegQIAhAC&url=https%3A%2F%2Faddons.mozilla.org%2Fen-GB%2Ffirefox%2Faddon%2Fublock-origin%2F&usg=AOvVaw0Bjb3P1OyQ5GKNBE_CFifL))
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage/subscribe
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are able to select a product as expected
- Compare to https://ubuntu-com-8294.demos.haus/advantage/subscribe, which looks broken while uBlock is active, and raises an error in the console that references ecom-analytics-events.js
